src/linker/Linker.Steps/SweepStep.cs

```diff
@@ -568,7 +568,9 @@ struct AssemblyReferencesCorrector
 		{
 			readonly AssemblyDefinition assembly;
 			readonly DefaultMetadataImporter importer;
+#pragma warning disable IDE0052 // Remove unread private members
 			readonly ITryResolveMetadata resolver;
```

Comment:
Do we still think the `resolver` is going to be useful for `AssemblyReferencesCorrector`? Maybe we could just remove the field.